tests/dashboard/test_product.py

```diff
@@ -542,7 +542,7 @@ def test_product_bulk_update_form_can_unpublish_products(product_list):
 
 def test_product_list_filters(admin_client, product_list):
     db_products = Product.objects.all()
-    assert len(db_products) == 2
+    assert len(db_products) == 3
```

Comment:
I'm not sure if this line is needed. Number of products bases on a fixture structure, which could be treated as a constant.

Reply:
hmmm looks like debuging assertion, deleted